kt/common/src/main/kotlin/godot/common/extensions/StringExtensions.kt

```diff
@@ -48,6 +48,48 @@ fun String.convertToSnakeCase(): String =
         else accumulator.append(character)
     }.toString()
 
+fun String.toScreamingSnakeCase(): String {
```

Comment:
Can you change this method name ? Like `toUpperSnakeCase` ?

Reply:
Done